core/src/global.rs

```diff
@@ -255,6 +255,7 @@ pub fn use_alternative_path(path_str: String) -> String {
 			check_path.display(),
 			p.display()
 		);
+		println!("The file `{}` was not found! Will try to use the alternative file `{}`!", check_path.display(), p.display());
```

Comment:
This comment is also for all occurrences in this commit. I think that it is better to use info!() or error!() to print such message on the epic-server.log.